cmd/terramate/cli/cli.go

```diff
@@ -112,6 +112,9 @@ type cliSpec struct {
 	DisableCheckpoint          bool `optional:"true" default:"false" help:"Disable checkpoint checks for updates"`
 	DisableCheckpointSignature bool `optional:"true" default:"false" help:"Disable checkpoint signature"`
 
+	Login struct {
+	} `cmd:"" help:"Login into Terramate Cloud" hidden:"true"`
+
```

Comment:
let's go with `experimental cloud login` for now ;)

Reply:
done!